src/skin/options.html

```diff
@@ -194,6 +194,17 @@ <h3><span class="i18n_export_user_data"></span></h3>
         <button id="exportTrackers"><span class="i18n_download"></span></button>
       </div>
     </div>
+    <div class="btn-silo">
+      <p class="i18n_sync_intro"></p>
+      <div id="upload">
+        <h3><span class="i18n_upload_whitelist"></span></h3>
+        <button id="uploadWhitelist"><span class="i18n_upload"></span></button>
```

Comment:
Should rename the IDs too ... "uploadWhitelist" to "cloud-upload"? (CSS names should be dash-separated and lowercase going forward.)